src/adapters/TelnetAdapter.ts

```diff
@@ -286,8 +287,7 @@ export class TelnetAdapter {
                     }
 
                     //watch for debugger prompt output
-                    // eslint-disable-next-line no-cond-assign
-                    if (match = /Brightscript\s*Debugger>\s*$/i.exec(responseText.trim())) {
+                    if (util.checkForDebuggerPrompt(responseText)) {
```

Comment:
Centralized the logic for detecting the debugger prompt so we didn't have the same(ish) regex in multiple spots.